docker-compose.yml

```diff
@@ -16,12 +16,13 @@ services:
       - db
 
   db:
-    image: postgis/postgis:9.6-2.5-alpine
+    build: db_docker
     container_name: db
     environment:
       - POSTGRES_PASSWORD=anyway
       - POSTGRES_USER=anyway
       - POSTGRES_DB=anyway
+      - DB_DUMP_PATH=/docker-entrypoint-initdb.d/anyway_public.pgdump
```

Comment:
Why not in /tmp/db.dump? The initdb.d directory should be used for init and init only